encord/dataset.py

```diff
@@ -132,6 +134,10 @@ def create_image_group(
             title:
                 The title of the image group. If unspecified this will be randomly generated for you. This title should
                 NOT include an extension. For example "encord_image_group".
+            video_image_groups:
+                Flag specifying image group representation, if True image group will be created with video
```

Comment:
How about:
```
A flag specifying how image groups are stored. If `True`, a compressed video will be created from the image groups. `True` was the previous default support. If `False`, the images are saved as a sequence of images.
```
Ideally - if we have some docs around this, we'd reference them here. It would be good for the user to understand tradeoffs. Do we have docs?